lighthouse-cli/bin.ts

```diff
@@ -44,6 +45,14 @@ interface LighthouseError extends Error {
 
 const cliFlags = getFlags();
 
+if (cliFlags.remoteDevice) {
+  cliFlags.port = defaults(cliFlags.port, 9222);
```

Comment:
I don't think this will work, since we set defaults for `port` and `disable-cpu-throttling` at least (to `0` and `false`, respectively), and `defaults()` checks for `undefined`.

I believe we can remove the `disable-cpu-throttling` default, though, since we can just use `undefined` as a falsy value in the normal case.